release-1.8/release_notes_draft.md

```diff
@@ -518,30 +537,35 @@ kind.
 
 #### kops
 
-* [alpha] Added support for targeting bare metal (or non-cloudprovider) machines. ([#360](https://github.com/kubernetes/features/issues/360), [@justinsb](https://github.com/justinsb)).
+* [alpha] Add support for targeting bare metal (or non-cloudprovider) machines. ([#360](https://github.com/kubernetes/features/issues/360), [@justinsb](https://github.com/justinsb)).
 
```

Comment:
Minor nit: I've been putting all these in past tense. But I can also touch this one in another PR if you like. lmk.